Testers/IntegrationTester/IntegrationTesterUITests/IntegrationTesterUITests.swift

```diff
@@ -459,7 +445,7 @@ class IntegrationTesterUITests: XCTestCase {
         completeAuth.forceTapElement()
 
         let safariOpenButton = safari.buttons["Open"]
-        XCTAssertTrue(safariOpenButton.waitForExistence(timeout: 5.0))
+        XCTAssertTrue(safariOpenButton.waitForExistence(timeout: 30.0))
```

Comment:
Why so long??

Reply:
This is the time between hitting "authorize", waiting for the PI to authorize, getting sent to the redirect URL, and getting the "Open in PaymentSheet Example?" popup from Safari. Combination of slow network, slowness on our side in testmode, and slow machines running the tests. It shouldn't take 30 seconds, but it definitely takes more than 5.